lib/api/src/rest/schema.rs

```diff
@@ -64,6 +64,19 @@ pub enum ShardKeySelector {
     // ToDo: select by pattern
 }
 
+fn vector_example() -> Option<VectorStruct> {
+    Some(VectorStruct::Single(vec![
+        0.875, 0.140625, -0.15625, 0.96875,
+    ]))
+}
+
+fn payload_example() -> Option<segment::types::Payload> {
+    Some(
+        serde_json::from_str(r#"{"city": "London", "color": "green"}"#)
+            .expect("Failed to parse payload"),
```

Comment:
You can use the [`json!`
](https://docs.rs/serde_json/latest/serde_json/macro.json.html) macro, which is a bit nicer (untested):

```suggestion
        serde_json::json!({
            "city": "London",
            "color": "green",
        })
```